.ci/jenkins/dsl/jobs.groovy

```diff
@@ -155,8 +108,19 @@ void setupDroolsJob(String droolsBranch) {
     }
 }
 
-void setupNightlyJob() {
-    def jobParams = KogitoJobUtils.getBasicJobParams(this, 'optaplanner-nightly', Folder.NIGHTLY, "${jenkins_path}/Jenkinsfile.nightly", 'Optaplanner Nightly')
+void setupProjectInitBranchJob() {
+    def jobParams = KogitoJobUtils.getBasicJobParams(this, '0-init-branch', Folder.INIT_BRANCH, "${jenkins_path_project}/Jenkinsfile.init-branch", 'Kogito Init Branch')
```

Comment:
It should not be `Kogito Init Branch`, but `OptaPlanner` would interfere with the lower-level init branch job.
Maybe "OptaPlanner Project Init Branch"?

Reply:
yeah forgot to replace after copy/paste ...
```suggestion
    def jobParams = KogitoJobUtils.getBasicJobParams(this, '0-init-branch', Folder.INIT_BRANCH, "${jenkins_path_project}/Jenkinsfile.init-branch", 'Optaplanner Project Init Branch')
```

It would not interfere as this is only the job description